### PR TITLE
Implement dark mode for reg-lite widget

### PR DESCRIPTION
**Pull request by tomrndom**
ref: https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskProject&view=vertical&task=3351853&stafilter=NotComplete&fileview=grid

Signed-off-by: Tomás Castillo <tcastilloboireau@gmail.com>

### CODE REVIEW

File: src/components/login-passwordless/index.module.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@
     .logo {
       width: 175px;
       align-self: center;
+      background-color: #fff;
```
Comment (smarcet) [inline src/components/login-passwordless/index.module.scss:19]: remove this

File: src/components/stripe-form/index.js [hunk 2]
```diff
@@ -78,10 +78,10 @@ const StripeForm = ({ reservation, payTicket, userProfile, marketingData, stripe
     const stripeStyle = merge({}, {
         base: {
             // Add your base input styles here. For example: #d4e5f4
-            color: marketingData.color_text_dark,
+            color: marketingData.color_input_text_color,
```
Comment (smarcet) [inline src/components/stripe-form/index.js:81]: what about is not defined ?

File: src/components/stripe-form/index.js [hunk 3]
```diff
@@ -78,10 +78,10 @@ const StripeForm = ({ reservation, payTicket, userProfile, marketingData, stripe
...
             fontSize: '16px',
             //fontFamily: 'inherit',
-            backgroundColor: '#ffffff',
+            backgroundColor: marketingData.color_input_background_color,
```
Comment (smarcet) [inline src/components/stripe-form/index.js:84]: same here

File: src/components/stripe-form/index.js [hunk 4]
```diff
@@ -78,10 +78,10 @@ const StripeForm = ({ reservation, payTicket, userProfile, marketingData, stripe
     const stripeStyle = merge({}, {
         base: {
             // Add your base input styles here. For example: #d4e5f4
-            color: marketingData.color_text_dark,
+            color: marketingData.color_input_text_color || marketingData.color_text_dark,
             fontSize: '16px',
             //fontFamily: 'inherit',
-            backgroundColor: '#ffffff',
+            backgroundColor: marketingData.color_input_background_color || '#ffffff',
```
Comment (smarcet) [inline src/components/stripe-form/index.js:84]: same here

**Review by smarcet — CHANGES_REQUESTED** [review]
@tomrndom please review comments

**Review by S-code652 — APPROVED** [review]

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**